src/components/Register/RegisterView.tsx

```diff
@@ -1,98 +1,108 @@
-
-                                        <div className="form-outline mb-4">
-                                            <input type="text"
-                                                   className="form-control form-control-lg"
-                                                   onChange={handleFormChange}
-                                                   name="firstName"
-                                                   value={formUser.firstName}
-                                                   placeholder="First Name"
-                                            />
-                                        </div>
-
-                                        <div className="form-outline mb-4">
-                                            <input type="text"
-                                                   className="form-control form-control-lg"
-                                                   onChange={handleFormChange}
-                                                   name="lastName"
-                                                   value={formUser.lastName}
-                                                   placeholder="Last Name"
-                                            />
-                                        </div>
-
-                                        <div className="form-outline mb-4">
-                                            <input type="email"
-                                                   className="form-control form-control-lg"
-                                                   onChange={handleFormChange}
-                                                   name="email"
-                                                   value={formUser.email}
-                                                   placeholder="Your Email"
-                                            />
-                                        </div>
+const RegisterView: FC<RegisterViewProps> = ({
+  handleSubmit,
+  handleFormChange,
+  formUser,
+}) => (
+  <div className="vh-100 bg-image">
+    <div className="mask d-flex align-items-center h-100 gradient-custom-3">
+      <div className="container h-100">
+        <div className="row d-flex justify-content-center align-items-center h-100">
+          <div className="col-12 col-md-9 col-lg-7 col-xl-6">
+            <div className="card" style={{ borderRadius: '15px' }}>
+              <div className="card-body p-5">
+                <h2 className="text-uppercase text-center mb-5">
+                  Create an account
+                </h2>
+                <form onSubmit={handleSubmit}>
+                  <div className="form-outline mb-4">
+                    <input
+                      className="form-control form-control-lg"
+                      name="firstName"
+                      onChange={handleFormChange}
+                      placeholder="First Name"
+                      type="text"
+                      value={formUser.firstName}
```

Comment:
Create separate component for form, create custom hook useForm for updating logic reusabitily.